catalystwan/session.py

```diff
@@ -301,30 +325,28 @@ def restart_imminent(self, restart_timeout_override: Optional[int] = None):
             self.restart_timeout = restart_timeout_override
         self.state = ManagerSessionState.RESTART_IMMINENT
 
-    def login(self) -> ManagerSession:
-        """Performs login to SDWAN Manager and fetches important server info to instance variables
-
-        Raises:
-            SessionNotCreatedError: indicates session configuration is not consistent
-
-        Returns:
-            ManagerSession: (self)
-        """
+    def _sync_auth(self) -> None:
         self.cookies.clear_session_cookies()
-        self._auth.clear()
+        self._auth.clear_sync(self._last_request)
         self.auth = self._auth
```

Comment:
I don't understand this line. Why we want make auth public after login. Do we need later?

Reply:
Yes, we need it. Auth is called with every request and `requests` library expects it in that place.

Reply:
We need it for requests :/